### ISSUE TITLE
Information symbol for components

### ISSUE THREAD

**Issue by groadybones** [SibylLab/program-wars#501]
**Is your feature request related to a problem? Please describe.**
With the move to the UI having a grouping field, main field, conditionals list, and player info panel, it might be a good to have a small inconspicuous symbol in the corner of each main component that when clicked, outputs the primary function/purpose of that component to remind the user of how to use that piece of the game without having to take them out of the action.

**Describe the solution you'd like**
I'm thinking the symbol should be a semi-transparent 'i' inside a circle as its a symbol commonly used for this exact purpose in lots of phone apps and web pages so users may already know that it will offer information when clicked even if they've never played and in other software it usually appears in the upper or lower right corner depending on the component its describing so we should probably follow suit to maintain user familiarity

**Comment by strinsberg** [comment]
I have started by adding an Icon that will show some information in a tool-tip when the mouse is above it. I have attached a screenshot to preview the changes.

![info_tooltips](https://user-images.githubusercontent.com/45857986/81110071-47636d80-8ed8-11ea-9cd5-4338b4ad2845.png)

Currently, there is only temporary text in the tool-tips. I will write something more specific if this design is acceptable.

I am not sure if it is necessary to have as many info icons as I added, or if the positions are perfect. The image for the icon can easily be changed to something else if desired.

If the image is not enough the current changes can be viewed in the componentInfo-#501 branch.

If you could take a look @johnanvik and let me know what you think.

**Comment by strinsberg** [comment]
If we wanted more information that might fit in a tool-tip we could set up the icons to open a modal with a detailed explanation when clicked. This could keep some concise help info in the tool-tip, but still allow a more detailed explanation to be looked at "without having to take [the user] out of the action" as mentioned in the original issue.

**Comment by strinsberg** [comment]
I played around with adding the ability to click the information icon. It seems like it would be difficult to put much useful information into the tooltip. Here is a screenshot of what it looks like.

![info_modal](https://user-images.githubusercontent.com/45857986/81209335-a3d49480-8f8d-11ea-9fad-a3f3c8bf1e4a.png)

I think the major question is how many info icons we actually want? I put one everywhere I thought it might be helpful, but I think there are too many right now. We don't want to clutter the UI to much. I suppose we could just put one or two on the player info pane and group information for the elements.

**Comment by johnanvik** [comment]
What if there was a checkbox or menu options to turn them all off and on? Then the user can turn it on if and when they want. A side benefit from a research/development point of view is that we could collect information on which icons are most commonly clicked. This would give insight into what the users find confusing.

**Comment by strinsberg** [comment]
I made all the info images subject to the tutorial checkbox. Now if a player is comfortable with the game and turns of the tutorial they will no longer see the icons.

I also grouped some of them together to make things a little less cluttered. I removed the one for the draw redraw buttons and just gave them tooltips. This is because there is not much info needed for each button, and the discard button is being moved in issue #429.

Full information text is also added now.

I will create a pull request for review of the changes and code and we can go from there.
